apps/api/v2/src/modules/organizations/routing-forms/controllers/organizations-routing-forms-responses.controller.ts

```diff
@@ -37,7 +38,7 @@ import { UpdateRoutingFormResponseOutput } from "../outputs/update-routing-form-
   path: "/v2/organizations/:orgId/routing-forms/:routingFormId/responses",
   version: API_VERSIONS_VALUES,
 })
-@UseGuards(ApiAuthGuard, IsOrgGuard, RolesGuard, PlatformPlanGuard, IsAdminAPIEnabledGuard)
```

Comment:
Moved RolesGuard to indvidual routes because Response creation endpoint needs to use RolesGuard along with isUserRoutingForm guard